extensions/standard-processors/processors/PutUDP.cpp

```diff
@@ -90,7 +90,7 @@ void PutUDP::onTrigger(core::ProcessContext& context, core::ProcessSession& sess
     for (const auto& resolver_entry : resolved_query) {
       error.clear();
       udp::socket socket(io_context);
-      socket.open(resolver_entry.endpoint().protocol(), error);
+      std::ignore = socket.open(resolver_entry.endpoint().protocol(), error);
```

Comment:
not related to this PR but we could check if our asio usage could use ASIO_NO_DEPRECATED=1, we seem to set it for couchbase